src/Databases/Properties/Number.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+namespace Notion\Databases\Properties;
+
+/**
+ * @psalm-type NumberJson = array{
+ *      id: string,
+ *      name: string,
+ *      type: "number",
+ *      number: array{ format: string },
```

Comment:
`self::FORMAT_*` (you'll start seeing this more and more as you start applying it :) )